src/libraries/System.Text.Json/src/System/Text/Json/Serialization/WriteStack.cs

```diff
@@ -50,7 +50,7 @@ public void Push(JsonClassInfo nextClassInfo, object nextValue)
             if (classType == ClassType.Enumerable || nextClassInfo.ClassType == ClassType.Dictionary)
             {
                 Current.PopStackOnEndCollection = true;
-                Current.JsonPropertyInfo = Current.JsonClassInfo.PolicyProperty;
+                Current.JsonPropertyInfo = Current.JsonClassInfo.PolicyProperty!;
```

Comment:
Isn't `WriteStackFrame.JsonPropertyInfo` nullable and so is `PolicyProperty`? Why do we need the `!` here?

Reply:
Right, seems it was annotated before making `WriteStackFrame.JsonPropertyInfo` nullable